packages/api/src/utils/client/httpClient.ts

```diff
@@ -269,7 +276,7 @@ export class HttpClient implements IHttpClient {
     const timer = this.metrics?.requestTime.startTimer({routeId});
 
     try {
-      const url = urlJoin(baseUrl, opts.url) + (opts.query ? "?" + stringifyQuery(opts.query) : "");
+      const url = new URL(urlJoin(baseUrl, opts.url) + (opts.query ? "?" + stringifyQuery(opts.query) : ""));
```

Comment:
This function `stringifyQuery` uses a third-party library. See if we can replace it with the native `node:querystring` module.

Reply:
We can't as it does not support all format required to be openapi compliant.

Specifically we want to "repeat" format like this `topic=topic1&topic=topic2`, although I think comma-separated should be supported by all clients as well

https://github.com/ChainSafe/lodestar/blob/f9c71074df49c4bdd1b01d8ac434719fda1a1dbf/packages/api/src/utils/client/format.ts#L5-L7

Reply:
As another side note, we use the same library for parsing, last time I looked at this, there was no native alternative for it and `qs` was the best library I could find

https://github.com/ChainSafe/lodestar/blob/1ff56f6ab3b74fa9998639f0f6c8773fb8181078/packages/beacon-node/src/api/rest/base.ts#L51-L58